riscv-elf.md

```diff
@@ -267,6 +280,31 @@ The ILP32E calling convention is not compatible with ISAs that have registers
 that require load and store alignments of more than 32 bits. In particular, this
 calling convention must not be used with the D ISA extension.
 
+## <a name=vector-calling-convention></a> Vector Calling Convention
+
+The vector calling convention provides sixteen argument registers, v8-v23, the
+first eight of which are also used to return values. How many vector registers
+will be used as the return value is depended on LMUL value of the return type.
+v0 is a special register for the mask in masked vector instructions.
+
+Vectors that are LMUL = 1 or fractional LMUL are passed in a single vector
+argument register. Vectors that are LMUL = 2 are passed in 2-aligned vector
+argument registers. Vectors that are LMUL = 4 are passed in 4-aligned vector
+argument registers. Vectors that are LMUL = 8 are passed in 8-aligned vector
+argument registers. If there is no available vector registers, vectors are
+passed by reference. If there are mask arguments, the first mask argument
+is passed in v0. The remaining mask arguments follow the rule for general vector
+arguments. The mask argument occupies a single vector register regardless how
+many bits are effective in the mask. If there is no available vector registers
+for the mask arguments, mask arguments are passed by reference. The value of
+mask arguments occupy VLEN bits on the stack.
+
+No vector registers shall be preserved across procedure calls.
+
+There is no scalar values passed through vector registers. There is no vector
+values passed through scalar registers. We have no need to define a new ABI
+for vector. Vector calling convention is appliable for existing ABIs.
+
```

Comment:
Not sure should we add more word about we are intending to having another calling conversion in future?